### ISSUE TITLE
Toolbar icons shrink too much

### ISSUE THREAD

**Issue by K-Kumar-01** [accordproject/web-components#228]
# Discussion 🗣

The icons in the toolbar area in the markdown editor seem to be too small on certain window sizes.
This doesn't look good. (the tooltip is shown on hover though)

## Context

Attaching an image for reference.
![DeepinScreenshot_20201218181333](https://user-images.githubusercontent.com/59891164/102616185-2c8af400-415d-11eb-9b9c-ab9d45c28162.png)
As seen above the toolbar icons are not visible.

## Detailed Description

The styling needs to be improved in my opinion.

**Comment by K-Kumar-01** [comment]
@irmerk 
I would like to work on this issue. Please assign this to me.

**Comment by K-Kumar-01** [comment]
@irmerk 
I have created a PR for the issue. Let me know if there are any changes required.
